src/styles/layout/_testimonial.scss

```
@@ -91,4 +91,54 @@
         margin-top: px-to-rem(117);
         margin-left: px-to-rem(130);
     }
+
+    &__btn {
+        height: 64px;
+        width: 64px;
+        position: absolute;
+        top: 120%;
+        border: 1px solid $midnight-express-10;
+        border-radius: 50%;
+        background-color: $white;
+        i[class^='icon-'] {
+            color: $black;
+        }
+        @include media-query('md') {
+            top: 45%;
+            height: 100px;
+            width: 100px;
+        }
+        &--next {
+            right: 0;
+
+            @include media-query('md') {
+                transform: translate(0px, px-to-rem(10));
```

```suggestion
                transform: translate(0, px-to-rem(10));
```
We don't use px with 0. CHeck and update at other places as well